### ISSUE TITLE
Kirby Image Blocks crashes Betterrest

### ISSUE THREAD

**Issue by Jones-S** [robinscholz/better-rest#20]
Hey unfortunately me again.
I just realized that for pages where I use Kirby Blocks with an image block the betterrest seems to crash. No data is returned whatsoever... :(

<img width="689" alt="image" src="https://user-images.githubusercontent.com/7059580/117073038-a947f480-ad31-11eb-8918-436897330dc2.png">
_Using an image block leads to no data returned..._

```json
{
  "status": "error",
  "exception": "Error",
  "code": 0,
  "message": "Call to a member function srcset() on null",
  "details": null,
  "file": "Betterrest.php",
  "line": 180
}
```

I would like to fix this issue but don't really know where to start.
If you could give me some pointers on that that would be great...

**Comment by Jones-S** [comment]
I started to investigate and already wanted to fork and commit a PR to change this:
```
// it is an array. if it is an image...
            if (\Kirby\Toolkit\A::get($value, 'type') === 'image') {
```
            
And then I realized that you already fixed this 3 months ago. lol.
Sorry for bothering. I close.

**Comment by Jones-S** [comment]
Oh but I reopen because there is no official release for composer installation containing the fix.
If you could make a new release that would be great! 👍

**Comment by robinscholz** [comment]
Just released 1.3 on packagist!
